Composer/packages/client/src/pages/design/index.tsx

```diff
@@ -118,6 +118,11 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
     }
   }, [dialogId, dialogs, location]);
 
+  useEffect(() => {
+    const index = currentDialog.triggers.findIndex(({ type }) => type === SDKKinds.OnBeginDialog);
+    ~index && selectTo(createSelectedPath(index));
```

Comment:
The `~index &&` is a cute trick, but I'd rather see something more legible like `if (index >= 0)`.